docker_volume_watcher/container_monitor.py

```diff
@@ -24,8 +28,8 @@ def docker_bind_to_windows_path(path):
         str:  Converts Hyper-V mount path to Windows path (e.g. /C/some-path -> C:/some-path).
 
     """
-    expr = re.compile('^(?:/host_mnt)?/([a-zA-Z])/(.*)$')
-    match = re.match(expr, path)
+    expr = re.compile(r'^(?:/host_mnt/|/run/desktop/mnt/host/)?([a-zA-Z]):?[/\\](.*)$')
```

Comment:
Why not `^(?:/host_mnt|/run/desktop/mnt/host)?/([a-zA-Z]):?[/\\](.*)$`?. Otherwise we will break compatibility with old versions (e.g. expression in your PR does not match `/C/some-path`)